airflow/providers/google/cloud/utils/credentials_provider.py

```diff
@@ -350,6 +365,24 @@ def _get_credentials_using_credential_config_file(self):
 
         return credentials, project_id
 
+    def _get_credentials_using_credential_config_file_and_token_supplier(self):
+        self._log_info(
+            "Getting connection using credential configuration file and external Identity Provider."
+        )
+
+        if not self.credential_config_file:
+            raise AirflowException(
+                "Credential configuration is needed to use authentication by External Identity Provider."
+            )
```

Comment:
I think it'd be very nice if the error message also explicitly provided an attribute name that must be supplied here, so users were able to figure it out without looking into the implementation.

Reply:
Sorry, but this comment isn't addressed.

Reply:
I believe the change you suggested has been addressed [here](https://github.com/apache/airflow/pull/39873/files#diff-da381f09f5f872e927033b8a3ecd43215dde198d956af677ac425b97603020e0R382). However, if I've misunderstood any part of your suggestion, please let me know.

Reply:
Ok, thank you